lib/sidekiq/scheduled.rb

```diff
@@ -47,7 +47,10 @@ def terminate
       private
 
       def zpopbyscore(conn, keys: nil, argv: nil)
-        @lua_zpopbyscore_sha = conn.script(:load, LUA_ZPOPBYSCORE) if @lua_zpopbyscore_sha.nil?
+        if @lua_zpopbyscore_sha.nil?
+          raw_conn = conn.respond_to?(:redis) ? conn.redis : conn
```

Comment:
I wonder if there's a better way to detect the use of `Redis::Namespace` here? I couldn't do `is_a?(Redis::Namespace)` because that is not a required depedency.

Reply:
What about `conn.class.name == "Redis::Namespace"`?